cmd/siklu/common_boot.c

```diff
@@ -36,9 +36,16 @@ char *kernel_path(void)
 		return BOOT_DIR "/zImage";
 }
 
+char *kernel_fit_path(void) {
+	return BOOT_DIR "/fitImage";
+}
+
+
 char *dtb_load_address(void)
```

Comment:
I don't think that the address 0 is correct.

Reply:
It's so we won't load from NULL, there's the same fix in the ctu branch